### ISSUE TITLE
Mulit label support

### ISSUE THREAD

**Issue by DecentMakeover** [Cadene/pretrained-models.pytorch#160]
Hi 

Thanks for sharing your work, is there support for multi-label image classification ?

Thanks

**Comment by dlazerka** [comment]
Google "multi hot encoding".

E.g:  just convert `last_layer` to `nn.Linear(1000, num_labels)`.
And provide targets (outputs) as a vector of num_labels length (multihot_encoded).

**Comment by DecentMakeover** [comment]
Thanks , will look into it.
